docs/install/local/README.md

```diff
@@ -109,13 +105,27 @@ If you select this option, it will ask if you want to run the service as the
 current user, or create a new `flowforge` user. If you choose to create the
 user, it will also change the ownership of the FlowForge directory to that user.
 
-### Configuring FlowForge
+## Configuring FlowForge (optional)
 
-The default FlowForge configuration is provided in the file `/opt/flowforge/etc/flowforge.yml`.
+The default FlowForge configuration is provided in the file `flowforge.yml`
+* Linux/MacOS: `/opt/flowforge/etc/flowforge.yml`
+* Windows: `c:\flowforge\etc\flowforge.yml`
 
 For more details on the options available, see the [configuration guide](../configuration.md).
 
-### Running FlowForge
+## Before running FlowForge (Windows Only)
```

Comment:
If we're singling this out, then we should be consistent and provide this warning for all OS as there might be things using the ports we default to using.

Or we update the install script to make this check - otherwise this is adding a lot of complication to the install for a 'just in case' scenario.

Or we change the default `start_port` to be high enough to not be a concern on any OS and remove the need for any of this.

Reply:
> we should be consistent and provide this warning for all OS

If I follow this route, do you know necessary Linux/MacOS command to view reserved ports?
Does Linux/MacOS have such a concept of ranges of reserved ports?

<br>

> Or we update the install script to make this check

That is possible but would require us asking (and explaining) to users how to run an administrative account.  
Also, would we need to do the same to the Linux/MacOS script?


<br>

> Or we change the default start_port to be high enough to not be a concern on any OS and remove the need for any of this.

Also possible, but there are a fair number of places where this is written verbatim (i think) - but certainly do-able


---

Please advise.

Reply:
The only "reserved" ports on Linux are those less than 1024 which normally only root can open, but there is no other concept of reserved ports.

Reply:
Notes from chat on this topic:

 - [ ] Update localfs to set start point to `12080` - @hardillb 
 - [ ] Update flowforge.yml to document 12080 - and config page in docs - @hardillb 
 - [ ] Update PR to remove the section about docs - @Steve-Mcl 
 - [ ] Update install readme remove the section on windows ports - @Steve-Mcl

Reply:
Info RE WSL + ports for future reference: `https://github.com/microsoft/WSL/issues/5514`

Reply:
I'm still seeing the section on Windows ports/Hyper-V etc. The reason for changing our default port to 12080 was to remove the need for this section in the install guide. This type of detail can go in the configuration docs - it is not needed here.

Reply:
weird - it is gone in local!?!?

Gonna re-check

Reply:
That was freaky, no amount of `git pull` synced up my local env!

Ended up using `.` web vscode! 

Resolved in [9465992](https://github.com/flowforge/flowforge/pull/1249/commits/9465992b4947c8b919d66eec970d281aaf2888ff)